Eatery Blue/UI/Profile/ProfileLoginModelController.swift

```diff
@@ -14,8 +14,9 @@ protocol ProfileLoginModelControllerDelegate: AnyObject {
 
 }
 
-class ProfileLoginModelController: ProfileLoginViewController {
+class ProfileLoginModelController: ProfileLoginViewController, AttemptLogin {
 
+    private var firstView = true
```

Comment:
```suggestion
    private var firstView: Bool = true
```

Reply:
I remember you saying to me once if I instantiate a variable when declared I should let type inference do its job

Reply:
The main reason is to keep it consistent with the other properties that you have below. I usually let type inference do the work but only if the other properties use it as well. Otherwise, keep it consistent and add the type.